ui/lib/apps/ClusterInfo/components/InstanceTable.js

```diff
@@ -194,6 +194,9 @@ export default function ListPage() {
       minWidth: 150,
       maxWidth: 300,
       isResizable: true,
+      onRender: (node) => (
+        <Tooltip title={`${node.deploy_path}`}>{node.deploy_path}</Tooltip>
```

Comment:
`title={node.deploy_path}` is just fine. btw, you can use the `<TextWrap>{node.deploy_path}</TextWrap>` to wrap the content, you can find more examples in useColumns.tsx.

Reply:
btw, I think the version column needs tooltip as well:

![image](https://user-images.githubusercontent.com/1284531/80297744-a0385500-87b8-11ea-9e10-26aca31fa915.png)